.github/workflows/pull-request.yml

```diff
@@ -111,6 +111,9 @@ jobs:
       - name: Get branch name
         uses: Alfresco/alfresco-build-tools/.github/actions/get-branch-name@v1.36.0
 
+      - name: Save commit message
+        uses: alfresco-build-tools/.github/actions/get-commit-message/action.yml
```

Comment:
you need to put the tag (in this case @v1.36.0 ) when you use an external action